library/src/fy_library/flows/create_template_model_using_parsed_fy_py_file/main_fy.py

```diff
@@ -92,6 +95,10 @@ def __call__(self) -> BaseTemplateModel:
                 return CreateMethodTemplateModel_UsingParsedFyPyFile_Flow(
                     parsed_fy_py_file=self._parsed_fy_py_file,
                 )()
+            case ParsedFyPyFileKind.PROPERTY_SETTER:
+                return CreatePropertySetterTemplateModel_UsingParsedFyPyFile_Flow(
+                    parsed_fy_py_file=self._parsed_fy_py_file,
+                )()
```

Comment:
Move to library/src/fy_library/mixins/property/template_model/using_parsed_fy_py_file_fy__for_setter_fy.py